src/components/CategoryFilter.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+
+const CategoryFilter = ({ handleFilterChange }) => {
+  const Category = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  return (
+    <div>
+      <h2>Select book with specific category</h2>
+      <select
+        onChange={handleFilterChange}
+        name="category"
+        required
+      >
+        <option value="All">
+          All
+        </option>
+        {Category.map(category => (
```

Comment:
- [ ] Instead of creating an option separated from the `map` function, why don't you just add the "All" string to the categories array? This way you would have less code and make your code cleaner.